app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -112,29 +116,39 @@ class AuthRepositoryTest : ShouldSpec(
 		}
 
 		should("call StorageReference.putBytes WHEN updateUserAvatar called") {
-			every { referenceToUserAvatar.downloadUrl } returns mockk {
-				coEvery { await() } returns userPhoto
-			}
+			coEvery { referenceToUserAvatar.downloadUrl.await() } returns userPhoto
 
 			tested.updateUserAvatar(userPhoto)
 
 			verify { referenceToUserAvatar.putBytes(byteArrayOutputStream.toByteArray()) }
 		}
 
 		should("return Profile WHEN updateUserAvatar called") {
-			every { referenceToUserAvatar.downloadUrl } returns mockk {
-				coEvery { await() } returns userPhoto
-			}
+			coEvery { referenceToUserAvatar.downloadUrl.await() } returns userPhoto
 
 			tested.updateUserAvatar(userPhoto) shouldBe profile
 		}
 
 		should("return Profile WHEN userProfile called") {
-			every { referenceToUserAvatar.downloadUrl } returns mockk {
-				coEvery { await() } returns userPhoto
-			}
+			coEvery { referenceToUserAvatar.downloadUrl.await() } returns userPhoto
 
 			tested.userProfile() shouldBe profile
 		}
+
+		should("set display name WHEN updateUserName called") {
+			val slot = slot<UserProfileChangeRequest>()
+			val currentUser = mockk<FirebaseUser> {
```

Comment:
please be consistent with declaring mocks :) we should declare them in one way, like val currentUser: FirebaseUser = mockk{
...
}

Reply:
Done 😄

Reply:
Candidate to rule 😎